src/main/java/TypewiseAlert/AlertTargetSelector.java

```diff
@@ -0,0 +1,20 @@
+package TypewiseAlert;
+
+import java.lang.reflect.InvocationTargetException;
+
+import TypewiseAlert.TypewiseAlert.BreachType;
+
+public class AlertTargetSelector {

```

Comment:
Class AlertTargetSelector Open for modification ...for every type of receiver new TO_*******() operation introduced